### ISSUE TITLE
Discussion: Travis CI

### ISSUE THREAD

**Issue by patrickmineault** [merely-useful/py-rse#630]
I really enjoyed reading the chapter on testing. However, I was wondering whether in the long term it was a good idea to recommend Travis CI, given their acquisition by a financial firm and [recent back and forth as to whether they will continue to support free actions for open source projects](https://news.ycombinator.com/item?id=25338983). In theory, Travis CI can be largely replaced with Github actions.

**Comment by lwjohnst86** [comment]
Yes, we have exactly that discussion. At the time of writing this, Travis CI was still fine, but as with any technology, things evolved. In our next version we're planning on switching over to GitHub Actions. Thanks for the issue!!
